appengine_flexible/websockets/main_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gorilla/websocket"
+)
+
+func TestSocketHandler(t *testing.T) {
+	var err error
+
+	server := httptest.NewServer(http.HandlerFunc(socketHandler))
+	defer server.Close()
+	dialer := websocket.Dialer{}
+
+	conn, resp, err := dialer.Dial("ws://"+server.Listener.Addr().String()+"/ws", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	got := resp.StatusCode
+	want := http.StatusSwitchingProtocols
+	if got != want {
```

Comment:
Inline `got` declaration, and optionally `want`, too?

`if got := resp.StatusCode; got != want {`

Reply:
Done